modules/local/format_prodigal.nf

```diff
@@ -0,0 +1,34 @@
+process FORMAT_PRODIGAL {
+    tag "$meta.id"
+    label 'process_low'
+
+    conda (params.enable_conda ? "conda-forge::gzip=1.12" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/gzip:1.11 ':
+        'quay.io/biocontainers/gzip:1.11 ' }"
+
+    input:
+    tuple val(meta), path(eggnog)
```

Comment:
Why `eggnog`? Confusing.

Reply:
I used eggnog_table.nf as template but I forgot to change the names in some areas :) I changed it.